rnd/docker-compose.yml

```
@@ -44,11 +45,42 @@ services:
       - REDIS_PORT=6379
       - REDIS_PASSWORD=password
       - AUTH_ENABLED=false
+      - PYRO_HOST=0.0.0.0
+      - EXECUTIONMANAGER_HOST=execution_manager
+      - EXECUTIONSCHEDULER_HOST=execution_scheduler
     ports:
       - "8000:8000"
     networks:
       - app-network
 
+  execution_manager:
```

`ExecutionManager` is the top-level process, but this service houses the entire executor stack. So:
```suggestion
  executor:
```

---

updated